### PR TITLE
Generate nominators accounts

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Generate nominators stash and controllers accounts

### CODE REVIEW

File: scripts/generate-accounts/index.ts [hunk 1]
```diff
@@ -164,25 +164,30 @@ class Accounts {
...
-      console.log(
-          `Nominator ${i} controller account has been written to the ${controllerAccountFilename}`
-      );
+      const account = await this.generateNominatorAccount(i);
+      accounts.push(account);
     }
+    return accounts;
+  }
+
+  private async generateNominatorAccount(id) {
```
Comment (AndreiNavoichyk) [inline scripts/generate-accounts/index.ts:177]: Reuse the validators method.

Comment (karthikeyan-cere) [reply scripts/generate-accounts/index.ts]: done

File: scripts/generate-accounts/index.ts [hunk 2]
```diff
@@ -106,18 +106,29 @@ class Accounts {
...
+    const number = +process.env.NOMINATOR_AMOUNT;
+    console.log(`Generating Nominator accounts of ${number}`);
+    for (let i = 1; i <= number; i++) {
+      const account = await this.generateStashControllerAccounts(i, 'nominator');
+      accounts.push(account);
+    }
+    return accounts;
+  }
+
+  private async generateStashControllerAccounts(id: number, name: string) {
```
Comment (AndreiNavoichyk) [inline scripts/generate-accounts/index.ts:120]: Rename to `Stash**And**Controller`

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/generate-accounts/index.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/generate-accounts/index.ts]: Updated.

**Pull request merged**